hack/seccomp.pl

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
```

Comment:
If you want I can help you rewrite this on go and use `go generate` for generating. So, developers won't be dependent on golang.